bluetooth_mesh/models/models.py

```diff
@@ -1592,17 +1592,18 @@ async def set_lightness_unack(
             retransmissions: int = 6,
             send_interval: float = 0.075,
     ) -> None:
-        tid = next(self.__tid)
+        tid = self.tid()
         remaining_delay = delay
 
         async def request():
+            nonlocal remaining_delay
```

Comment:
Right, I keep forgetting about that keyword... Would you be kind enough to refactor `values` dict used in other models?

Reply:
`values` is a dict, so it's mutable by default.

Reply:
Yeah, and I've used `dict` because I forgot about `nonlocal` keyword, even though the code is modifying just one key in that dict. I like your approach better.